share/skel/cpanfile

```diff
@@ -1,9 +1,37 @@
 requires "Dancer2" => "[d2% dancer_version %2d]";
 
-recommends "YAML"             => "0";
-recommends "URL::Encode::XS"  => "0";
-recommends "CGI::Deurl::XS"   => "0";
-recommends "HTTP::Parser::XS" => "0";
+recommends "YAML"                    => "0";
+recommends "URL::Encode::XS"         => "0";
+recommends "CGI::Deurl::XS"          => "0";
+recommends "HTTP::Parser::XS"        => "0";
+recommends "CBOR::XS"                => "0";
+recommends "YAML::XS"                => "0";
+recommends "Class::XSAccessor"       => "0";
+recommends "Cpanel::JSON::XS"        => "0";
```

Comment:
Since we have a dep on `JSON::MaybeXS` then this `recommends` feels wrong.

Reply:
Leaving this one per our conversation. We also have this `recommends` in the project's `cpanfile`.